src/app/reimbursements/page.tsx

```diff
@@ -46,17 +39,13 @@ const formSchema = z.object({
   paymentMethod: z.string().min(1).max(100),
   purpose: z.string().max(1000),
   file: z.any(),
-  comment: z.string(),
 });
 
 export default function Page() {
   const { isLoaded, isSignedIn, user } = useUser();
   const [isConfirmed, setIsConfirmed] = useState(false);
-  const [uploadSuccess, setUploadSuccess] = useState(false);
-  const [submitDisabled, setSubmitDisabled] = useState(false);
-  const popupRef = useRef<{ openDialog: () => void } | null>(null);
-
```

Comment:
still looks like some earlier commits are getting overwritten? also could you check the ci script to see why it isn't building

Reply:
done! Added them back in but I don't believe they're needed the way it's currently written.